pkg/ocicni/ocicni.go

```diff
@@ -305,7 +305,9 @@ func loadNetworks(confDir string, cni *libcni.CNIConfig) (map[string]*cniNetwork
 
 		logrus.Infof("Found CNI network %s (type=%v) at %s", confList.Name, confList.Plugins[0].Network.Type, confFile)
 
-		networks[confList.Name] = cniNet
+		if _, ok := networks[confList.Name]; !ok {
+			networks[confList.Name] = cniNet
```

Comment:
@s1061123 can we log at debug level that ocicni is ignoring the duplicate file?

Reply:
Done.